prime-router/docs/how-to-setup-an-apple-silicon-mac.md

```diff
@@ -0,0 +1,91 @@
+# Setup Apple Silicon Macs for Development
+
+## Problems
+
+At the time of writing this note, Microsoft's Azure Function docker container is not compatible with Apple Silicon processors. 
+Microsoft has not announced plans to fix this problem. 
+
+The current `./devenv-infrastructure.sh` runs the Frontend build in a container. 
+This container doesn't run because of the use of ARM64. 
+
+## Workarounds
+
+Fortunately, most of our tooling works on Apple Silicon, and it is possible to run ReportStream without a Docker container.  
+
+### Step 1 - Follow the getting started instructions
+
+Follow most of the [Getting Started](getting_started.md) instructions, including:
+
+1. Installing the recommend tools 
```

Comment:
In my case I needed to install the tools and also update an entry in the `worker.config.json` file with the absolute path of my java install for the `defaultExecutablePath` key. For me the `worker.config.json` file was at `/opt/homebrew/opt/azure-functions-core-tools@3/workers/java/worker.config.json`, I think that will be consistent for homebrew installs but it may not.

This took care of the `did not find any initialized language workers ` related errors.